### ISSUE TITLE
Is it possible to support Waydroid as container?

### ISSUE THREAD

**Issue by batoceragaming** [uureel/batocera.pro#34]
I have tested installing the arch container, and I am full blown by the complete set. It's just amazing. In term of container, is it possible to support Waydroid in the future? supporting waydroid might unlock the potential of running android apps in Batocera.

I ask this because, Batocera has wayland. So in theory, running waydroid in Batocera could be a thing in the future.

Thanks for your awesome work.
:)

**Comment by trashbus99** [comment]
Nope.  Batocera uses X11/Xorg, not Wayland which Waydroid requires.

**Comment by uureel** [comment]
bluntly, it wouldn't be much worth supporting waydroid, bato currently can't do wayland for x86 and for good reasons too (for one doesn't work too well with nv, which is an optimistic way to put it), but even if wayland was there, waydroid still needs some lowlevel env/stuff that would be borky or altogether unobtainable in batocera, and then even if it were there, still, waydroid just isn't the sort of environment you could get some actual/real user-friendly-use out of in bato context (gamepad/pad2key/system automation etc); and then even if it did run through all of this, even with wayland and all, and even if you're fine using mouse and keyb and run configs and clickthroughs and troubleshoots and whatnot to get it going, then, still you're not even guaranteed to get proper gpu acceleration in it, eg with nv 

tldr, not much point apart from proof of concept; just as with windows games, if android is of any real priority - it's much better to just use retrobat
